components/articles/about.tsx

```diff
@@ -5,13 +5,13 @@ import {colors} from "../../styles/colors";
 
 export default function About(): JSX.Element {
```

Comment:
use ES6 arrow function syntax, for consistency with everything else but its also newer, cleaner, and super hip